example/reactnative/android/gradle.properties

```diff
@@ -27,12 +27,13 @@ android.useAndroidX=true
 # ./gradlew <task> -PreactNativeArchitectures=x86_64
 reactNativeArchitectures=armeabi-v7a,arm64-v8a,x86,x86_64
 
+# NOTE(louis): The Authgear SDK is not compatible with the new arch.
 # Use this property to enable support to the new architecture.
 # This will allow you to use TurboModules and the Fabric render in
 # your application. You should enable this flag either if you want
 # to write custom TurboModules/Fabric components OR use libraries that
 # are providing them.
-newArchEnabled=true
+newArchEnabled=false
```

Comment:
What will happen if we set `true` here?

Even our sdk is not using new arch, we might also want to ensure it can run in an app using new arch?